docs/topics/cards/physical/index.mdx

```diff
@@ -159,7 +159,7 @@ The printing hub is defined in the [card product ID](../index.mdx#card-product).
 If you'd like to ship cards out of both hubs, you need two card products: one with France as the hub, and the other with Spain.
 :::
 
-If your cardholder reports not receiving their physical card, please refer them to the [Swan Support Center](https://support.swan.io/hc/en-150/articles/5503032519837).
+If your cardholder reports not receiving their physical card, submit a request to [Swan Support](https://support.swan.io/hc/en-150).
```

Comment:
I couldn't find a relevant page to redirect to, so I've redirected to the Support portal (as mentioned [here](https://support.swan.io/hc/en-150/articles/17791219525149-Order-your-physical-card#:~:text=If%20it's%20been%20three%20weeks%20and%20you%20still%20haven't%20received%20your%20card%2C%20submit%20a%20request%20to%20Swan%20Support.%20If%20you%20are%20notified%20that%20your%20card%20is%20lost%20during%20delivery%2C%20please%20order%20a%20new%20card.)).